components/support/rc_crypto/src/lib.rs

```diff
@@ -2,8 +2,17 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/// This crate provides all the cryptographic primitives required by
+/// this workspace, backed by the NSS library.
+/// The exposed API is pretty much the same as the `ring` crate
+/// (https://briansmith.org/rustdoc/ring/) as it is well thought.
```

Comment:
Could you please also add a top-level `README.md` for the `rc_crypto` create, to help with future external review? The `viaduct` crate has a good example of the kind of scene-setting material that can be useful in such a doc. For this one, I imagine it saying a few words about why we use `NSS` and about the way the API is modelled after ring.